goldiloader.gemspec

```diff
@@ -24,8 +24,8 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3'
 
-  spec.add_dependency 'activerecord', '>= 4.2', '< 6.1'
-  spec.add_dependency 'activesupport', '>= 4.2', '< 6.1'
+  spec.add_dependency 'activerecord', '>= 4.2', '< 6.2'
+  spec.add_dependency 'activesupport', '>= 4.2', '< 6.2'
```

Comment:
Why `< 6.2` when `6.1` isn't out yet?

Reply:
Rails edge (i.e. `master`) has moved to version 6.1 already and we want to continue testing with that too.